src/components/Auth/AuthConfig.tsx

```diff
@@ -0,0 +1,174 @@
+    defaultCustomization.guidanceCustomization.buttonTextNormalColor =
+      uiDef.authButtonTextNormalColor
+    defaultCustomization.guidanceCustomization.buttonTextDisabledColor =
+      uiDef.authButtonTextDisabledColor
+    defaultCustomization.guidanceCustomization.buttonTextHighlightColor =
+      uiDef.authButtonTextHighlightColor
+    defaultCustomization.guidanceCustomization.buttonRelativeWidth = '1f'
+    defaultCustomization.guidanceCustomization.buttonCornerRadius =
+      uiDef.authButtonCornerRadius
+    defaultCustomization.guidanceCustomization.retryScreenImageBorderColor = color(
+      'neutral-white'
+    )
+    defaultCustomization.guidanceCustomization.retryScreenImageBorderWidth =
+      '2px'
+    defaultCustomization.guidanceCustomization.retryScreenImageCornerRadius =
+      '0'
+    defaultCustomization.guidanceCustomization.retryScreenOvalStrokeColor =
+      uiDef.authRetryScreenOvalColor
+    // defaultCustomization.guidanceCustomization.retryScreenSlideshowImages = retryScreenSlideshowImages
+    defaultCustomization.guidanceCustomization.retryScreenSlideshowInterval =
+      '1500'
+    defaultCustomization.guidanceCustomization.enableRetryScreenSlideshowShuffle = true
+    defaultCustomization.guidanceCustomization.enableRetryScreenBulletedInstructions = true
+
+    // Set Oval Customization
+    defaultCustomization.ovalCustomization.strokeColor = uiDef.authOvalColor
+    defaultCustomization.ovalCustomization.progressColor1 =
+      uiDef.authDualSpinnerColor
+    defaultCustomization.ovalCustomization.progressColor2 =
+      uiDef.authDualSpinnerColor
+
+    // Set Feedback Customization
+    defaultCustomization.feedbackCustomization.backgroundColor =
+      uiDef.authFeedbackBarColor
+    defaultCustomization.feedbackCustomization.textColor =
+      uiDef.authFeedbackBarTextColor
+    defaultCustomization.feedbackCustomization.cornerRadius =
+      uiDef.authButtonCornerRadius
+    defaultCustomization.feedbackCustomization.relativeWidth = '1f'
+
+    // Set Result Screen Customization
+    defaultCustomization.resultScreenCustomization.backgroundColors =
+      uiDef.authFrameColor
+    defaultCustomization.resultScreenCustomization.foregroundColor =
+      uiDef.authTextColor
+    defaultCustomization.resultScreenCustomization.activityIndicatorColor =
+      uiDef.authAccentColor
+    defaultCustomization.resultScreenCustomization.resultAnimationBackgroundColor =
+      uiDef.authAccentColor
+    defaultCustomization.resultScreenCustomization.resultAnimationForegroundColor =
+      uiDef.authFrameColor
+    defaultCustomization.resultScreenCustomization.uploadProgressFillColor =
+      uiDef.authAccentColor
+
+    // Set Animated Customization
+    const loaderSVG = document.createElementNS(
+      'http://www.w3.org/2000/svg',
+      'svg'
+    )
+    loaderSVG.setAttribute('viewBox', '0 0 56 56')
+    loaderSVG.classList.add('experiment-svg')
+    loaderSVG.innerHTML = loader
```

Comment:
I did a `loaderSVG.innerHTML = sanitize(loader)` and unfortunately it doesn't show anything where the loader SVG was supposed to be :(

Reply:
Is `loader` a string or a node?

Reply:
For some reason my previous comment disappeared, so pasting it here again
If `dompurify` doesn't work with this then we should bring this up with security because I'm pretty sure this is high risk.

Reply:
it's a string! I tried printing the "sanitized" value and whereas `loader = '<path....'`, `sanitize(loader) = ''`

Reply:
Security team gave us the green light on this so I will resolve this conversation